continue-reading.html

```diff
@@ -1,13 +1,231 @@
 <!DOCTYPE HTML>
 <html>
-  <head>
-    <title>Continue Reading</title>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
-  </head>
-  <body>
-    <section id="header"></section>
-    <section id="main"></section>
-    <section id="footer"></section>
-  </body>
+
+<head>
+  <title>Motherflippin Homepage</title>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+  <link href="style.css" rel="stylesheet" />
+</head>
+
+
+<body id="container">
+
+
+
+

```

Comment:
Using the same `id="container"` on multiple pages will cause issues down the line. I've noticed that you reuse a lot of class names and id names. If you were to go through and properly style your "Continue Reading" page. When you look at your other pages, your css won't line up due to the content of each page being different. 

One way to work around this is to have your body be something like `id="continue-container"` (or something similar), and naming that each way for each page. Then doing specific CSS on each page so you can be more specific about what you're targeting and adjusting.